torch/distributed/pipelining/_backward.py

```diff
@@ -271,15 +266,15 @@ def stage_backward_weight(
             else:
                 weight.grad += dw
     # return grads in the original order weights were provided in
-    return weight_grads
+    return tuple(weight_grads)
```

Comment:
these changes look pretty innocuous to me, but can you convince me that this change doesn't add any restriction or limitation to the user code?